deepchecks/base/dataset.py

```diff
@@ -198,10 +206,9 @@ def from_numpy(
         Raises:
             DeepchecksValueError:
                 if receives zero or more than two numpy arrays;
-                if features (args[0]) is not two dimensional numpy array;
+                if columns (args[0]) is not two dimensional numpy array;
                 if labels (args[1]) is not one dimensional numpy array;
                 if features array or labels array is empty;
```

Comment:
Why is this an error? labels array can be None...
(and did you mean to rename to "features")

Reply:
it's not about features & labels, `arg[0]` is the columns array so it must be 2 dimensional (columns X samples)

Reply:
oh got it...
then only grammar here (is not *a* two/one dimensional numpy array)